WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -1098,4 +1098,49 @@ def change_and_check_activities(new_start_date, new_end_date)
     expect(competition).not_to be_valid
     expect(competition.errors.messages[:organizer_ids].first).to match "Need a country"
   end
+
+  describe "exempt_from_wca_dues" do
```

Comment:
We don't do underscores in our spec names usually. These are supposed to be more human-readable. in this case it could be `describe "is exempt from Dues" do`

Reply:
Updated. Thank you!